### PR TITLE
Svelte workshop lp

### PR DESCRIPTION
**Pull request by yaigrie**
./svelte-sveltekit-workshop/
content from normal svelte workshop
template from old rust workshop lp

## TODO

- [x] add `og:image` incl. dates and times (see https://www.sketch.com/s/066e539c-c206-4261-aafb-9634ab577579/a/5nxJWJy for reference)
- [x] replace placeholder images (maybe you can find some here where someone from the team is giving a talk or workshop: https://drive.google.com/drive/folders/1SDTO9dh3JJbhk_CmILcMvR2m5Cie2hzP?usp=share_link and we might find some workshop situations or rooms on http://unsplash.com)
- [x] include Svelte logo in header like on https://mainmatter.com/svelte-consulting/

### CODE REVIEW

**Comment by marcoow** [comment]
@yaigrie I made some additional changes to the content and added the configuration for the subdomain. Can you complete the TODOs above? We can put this live on Monday 16th then.

**Comment by yaigrie** [comment]
@marcoow I added the og image and the svelte logo!
For the workshop images I chose two from the Vienna workshop, are those suitable?

**Comment by marcoow** [comment]
I think the photos from Vienna work 👍 Can you remove all photos that are not used anymore from this PR?

Also, please let's add the dates etc. of the workshop in the `og:image` – see the example I linked to above

File: static/assets/images/svelte-sveltekit-workshop/svelte-workshop-og-image.png
Comment (marcoow) [inline static/assets/images/svelte-sveltekit-workshop/svelte-workshop-og-image.png:1]: let's rearrange how we display the date:

> 5th – 7th December 2023, 14:00-18:00 CEST

**Pull request merged**